plutus-contract/src/Plutus/Contract/Request.hs

```diff
@@ -128,6 +130,14 @@ awaitSlot ::
     -> Contract w s e Slot
 awaitSlot s = pabReq (AwaitSlotReq s) E._AwaitSlotResp
 
+-- | Get the 'SlotConfig' which defines the length of 1 slot and the starting time.
+getSlotConfig ::
```

Comment:
The `SlotConfig` is pretty specific to the emulator, there isn't a direct equivalent to it when the PAB is connected with the real node & WBE. So it makes sense to have this function in `EmulatorConfig`, but I don't think we should actually have it in `Contract`.

Sorry, I know that's what the issue was about, but I had not fully thought it through.

Reply:
Ah ok! Then I'll remove `getSlotConfig` from `Contract`. Then, it won't be possible for a `Contract` to have access to the `SlotConfig` of the real node?

Reply:
> Some functions with Contract need to know the SlotConfig

Yeah it is best to pass the `SlotConfig` in from the outside, if required (for example with an endpoint or as a function `SlotConfig -> Contract`). This is one part of the node integration that I'm not 100% sure what it will look like in the end.